frontend/src/components/HealthMetrics.tsx

```diff
@@ -84,21 +86,23 @@ const HealthMetricsCharts: React.FC<{ data: HealthMetricsProps[] }> = ({ data })
         />
       </div>
       <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
-        <RadialChart
+        <GradientRadialChart
           title="Days Since Last Commit"
           icon={faCodeCommit}
-          series={data.map((item) => item.lastCommitDays)}
-          labels={labels}
+          days={data[0]?.lastCommitDays ?? 0}
+          requirement={data[0]?.lastCommitDaysRequirement ?? 0}
```

Comment:
While reviewing this, I noticed that it always returns 365 for any project I test with. This behavior is expected based on the current setup - it’s not a but - but we’ll need to adjust it to better reflect reality. Naturally, the number of releases and commits should differ between Flagship and Incubator projects.

I'll sync with Ark to clarify what the expected values should be, and once we have that direction, I’ll create a separate task to update the logic accordingly. 👍🏼